inclusive-menu-button.js

```diff
@@ -4,7 +4,26 @@
   'use strict'
 
   // Constructor
-  function MenuButton (button) {
+  function MenuButton (button, userSettings) {
+    // Error if the userSettings argument isn't an object
+    if (userSettings !== undefined && typeof userSettings !== 'object') {
+      throw new Error('MenuButton\'s second argument is expected as an object or undefined')
+    }
+
+    // The default settings
+    this.settings = {
+      radios: false
+    }
+
+    // Overwrite defaults where they are provided in userSettings
```

Comment:
If [support](http://kangax.github.io/compat-table/es6/#test-Object_static_methods_Object.assign) is okay, you could (should?) do:

```js
this.settings = Object.assign({
  radios: false
}, userSettings)
```